packages/main/src/styles/global.css

```diff
@@ -190,15 +190,15 @@ h6 {
   } */
 /* styles for notification modal */
 body > div:nth-child(7) > div > div > div > div > div > div >div >div{
-  width:400px
+  width:400px;
   }
-  body > div:nth-child(7) > div > div > div > div > div > div >div >div >div{
+body > div:nth-child(7) > div > div > div > div > div > div >div >div >div{
     background-color:white;
     }
-    body > div:nth-child(7) > div > div > div > div > div > div >div >div >div >div >button{ 
+body > div:nth-child(7) > div > div > div > div > div > div >div >div >div >div >button{ 
       background-color:#495057;
       } 
-      body > div:nth-child(7) > div > div > div > div > div > div >div >div >div >div >div{ 
+  body > div:nth-child(7) > div > div > div > div > div > div >div >div >div >div >div{ 
```

Comment:
Isn't there a way to use a className or id for this?

Reply:
The component is from a library and I have tried using classes to select it but I guess it is too nested

Reply:
this can be dangerous if it matches something else
try and add a data id or something specific